satellite/repair/repairer/repairer.go

```diff
@@ -29,6 +29,7 @@ type Config struct {
 	MaxRepair                     int           `help:"maximum segments that can be repaired concurrently" releaseDefault:"5" devDefault:"1"`
 	Interval                      time.Duration `help:"how frequently repairer should try and repair more data" releaseDefault:"1h" devDefault:"0h5m0s"`
 	Timeout                       time.Duration `help:"time limit for uploading repaired pieces to new storage nodes" default:"10m0s"`
+	DownloadTimeout               time.Duration `help:"time limit for downloading pieces from a node for repair" default:"2m0s"`
```

Comment:
Can we bump this up to at least 5m? I'd rather try for a little bit longer to get the pieces we need before giving up and repairing them and adding to the garbage load. It's fairly reasonable for a node to be under high load or high network congestion for 2 minutes.